Assets/Test_PlayMode/TestRunner_PlayMode.cs

```diff
@@ -34,7 +35,30 @@ public IEnumerator TestRunWithEnumeratorPasses()
             Scene scene = SceneManager.GetActiveScene();
 
             // asssrt
-            Assert.AreEqual(scene.name, "");
+            Assert.AreEqual(scene.name, "Main");
         }
+
+        public IEnumerator TestRunner_PlayModeWithEnumeratorPasses()
+        {
+            // Arrange
+            var obj = new GameObject();
+            GameManager gm = obj.AddComponent<GameManager>();
+            TimeSpan ts = new TimeSpan(0, 9, 50);
+
+            yield return null;
+
+            // act 
+            // 
+            DateTime startTime = DateTime.UtcNow;
+            do
+            {
+                yield return null;
+            } while  ((DateTime.UtcNow - startTime).TotalSeconds < 10.0);
```

Comment:
Play mode에서는 실제 game play 상황이므로 Time.deltaTime이 유효합니다.
그래서 아래와 같이 조금 직관적으로 코드 작성이 가능합니다.
```suggestion
            float waitTime = 0.0f;
            do
            {
                yield return null;
                waitTime += Time.deltaTime;
            } while (waitTime < 10.0f);
```

Reply:
생각해보니 Time.deltatime으로 한거 보여주셨었는데 까먹었었네요 ㅎㅎㅎ 
감사합니다~